### PR TITLE
DT-1512 username to be in lowercase(in email text)  when email is use…

### PR DESCRIPTION
**Pull request by dnyanesh-sonwane**
…rname

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/oauth2server/verify/ResetPasswordServiceImpl.kt [hunk 1]
```diff
@@ -182,7 +182,7 @@ class ResetPasswordServiceImpl(
 
   private fun sendPasswordResetEmail(user: User) {
     // then the reset token
-    val username = user.username
+    val username = if (StringUtils.contains(user.username, "@")) user.username.toLowerCase() else user.username
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/oauth2server/verify/ResetPasswordServiceImpl.kt:185]: kotlin does have a `contains` on `String` so don't need to call `StringUtils` for it

**Review by petergphillips — APPROVED** [review]
LGTM

File: src/main/kotlin/uk/gov/justice/digital/hmpps/oauth2server/verify/ResetPasswordServiceImpl.kt (hunk 1 above)
Comment (dnyanesh-sonwane) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/oauth2server/verify/ResetPasswordServiceImpl.kt]: Done

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**